pages/home/auth-providers/oauth2.mdx

```diff
@@ -30,17 +30,29 @@ The only supported OAuth 2.0 flow is the authorization code grant flow (with or
 
 How you configure the OAuth 2.0 provider depends on whether you use the Arcade Cloud Engine or a [self-hosted Engine](/home/install/overview).
```

Comment:
You must always use the cloud address as the redirect URL. Can we revert this change but add something here like:

```suggestion
How you configure the OAuth 2.0 provider depends on whether you use the Arcade Cloud Engine or a [self-hosted Engine](/home/install/overview). If you use the Cloud Engine, you must configure your provider in the Dashboard. 

<Note>
When configuring your app in the OAuth 2.0 enabled service, you must use `https://cloud.arcade.dev/api/v1/oauth/callback` as the redirect URL (sometimes called callback URL).
</Note>
```